doc/specification.md

```diff
@@ -84,11 +84,20 @@ To enable interchangeability, the files are characterized by
 The JSON files additionally follow
 * 2 spaces indentation
 
-## Format Compatibility
+## Format Versions and Compatibility
 
-The format of object types will evolve over time. If a change to the format is considered incompatible, the version of the file format has to be increased.
+The ABAP file format of object types will evolve over time.
+For this purpose the JSON schema specifies the field `formatVersion` and the file itself, is named after the version.
```

Comment:
Shouldn't it be `$formatVersion`? Sorry, if I missed the decision against `$`.

Reply:
I understood that @albertmink don't want to add the `$`. I was wondering whether it would help to indicate that `formatVersion` contains meta information about the object. However, both solutions would be fine for me.